web/src/features/fba/components/viz/ElevationInfoViz.tsx

```diff
@@ -19,9 +19,10 @@ const classes = {
 
 const Root = styled('div')({
   [`& .${classes.header}`]: {
```

Comment:
Do we need this styled div anymore?

Reply:
Good catch. We probably haven't needed this for quite a while. We don't need to render anything (including an empty div) when there is no elevation data. 
Fixed in https://github.com/bcgov/wps/pull/3792/commits/c06a9100674558a3c0586a75ed23d6f2f652cb33